### DISCUSSION TITLE
The Lengyel model for the divertor and SoL

### DISCUSSION THREAD

**Discussion by jmorris-uk** [ukaea/PROCESS#3046] [Ideas]
### Original issue (https://github.com/ukaea/PROCESS/issues/1381)
>
> The exhaust group (Stuart Henderson, David Moulton et al) have implemented the Lengyel model for the divertor, with enhancements. This is somewhat similar to the Kallenbach model (already implemented in PROCESS) but written by actual edge physicists and benchmarked against SOLPS.
>
> This model is available in Python from Stuart Henderson.
>
> I suggest we assess this as an alternative to the Kallenbach model.

### Papercut review

The model developed by M Kryjak is

>The model is called DLS-Extended and is an enhanced version of the DLS model which was originally written by Bruce Lipschultz and improved by Cyd Cowley (CCd, also see attached paper). It’s a simple Lengyel type model which as David says predicts the front location for given upstream conditions and equilibrium along a single field line.

It only considers impurity mechanisms for power loss in the SOL and doesn't account for recombination, electron impact ionisation, and charge exchange. The primary aim of the model appears to look at the achievement of detachment, its stability, and the location of the detachment front along the field lines.

The model is on GitHub but has no licence yet -- https://github.com/cydcowley/DLS-model

Publication on the DLS model is available here https://iopscience.iop.org/article/10.1088/1741-4326/ac7a4c

To review and implement this model is a large piece of work, add to the before beginning we would need to properly assess how it adds to the Kallenbach model and if the detachment information is useful for PROCESS.

It is possible that the 1-D model in the HERMES code might be better suited ([publication here](https://www.sciencedirect.com/science/article/pii/S0010465523003363)). 

**The original issue has been closed and discussion on this model moved here.**

**Comment by mkovari** [comment]
See also #1211, which mentions both SD1D and a Dutch code called DIV1D.

(now moved to discussion #3487)

**Comment by j-a-foster** [comment]
Another issue related to this was #1201.
